### PR TITLE
For #370: Forgot password post screen and logic

### PR DESCRIPTION
**Pull request by paulodamaso**
For #370: Forgot password post screen and logic
- Created test that checks if the page showed after forgot password post is right
- Created forgot password post screen
- Corrected the test in forgt password

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#372](https://github.com/timelesslounge/timelessis/issues/372) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#372](https://github.com/timelesslounge/timelessis/issues/372) is assigned to @ammaratef45[/z](https://www.0crat.com/u/ammaratef45), here is [why](https://www.0crat.com/footprint/CF9LG1YCA/8003d54f-f9ac-4598-8b99-5cbf48942bcf); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: timeless/auth/views.py [hunk 1]
```diff
@@ -58,12 +58,15 @@ def logout():
 @bp.route("/forgotpassword", methods=("GET", "POST"))
 def forgot_password():
     if request.method == "POST":
-        error = auth.forgot_password(email=request.form["email"])
+        email=request.form["email"]
```
Comment (ammaratef45) [inline timeless/auth/views.py]: @paulodamaso Missing whitespace around operator

File: timeless/auth/views.py [hunk 2]
```diff
@@ -58,12 +58,15 @@ def logout():
...
         if error is not None:
-            flash("We sent a password recover link to your email tst@mail.com")
-            return redirect(url_for("auth.login"))
+            return render_template(
+                "auth/forgot_password_post.html",
+                email=email
+            )
         else:
-            flash("E-mail not found", "error")
+            return render_template("auth/forgot_password.html", error=error)
```
Comment (ammaratef45) [inline timeless/auth/views.py]: @paulodamaso `error` is `none` here, right? shouldn't you replace the body of if and else?

File: timeless/templates/auth/forgot_password_post.html [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{% extends 'base.html' %}
+
+{% block header %}
+  <h1>{% block title %}Forgot password{% endblock %}</h1>
+{% endblock %}
+{% block content %}
+  <h1>Forgot your password?</h1>
+  <p>We've sent an e-mail to {{email}} with your new password.</p>
+  <p>Please use it to log in and change it.</p>
+{% endblock %}
```
Comment (ammaratef45) [inline timeless/templates/auth/forgot_password_post.html]: @paulodamaso error is not used here!?

**Comment by ammaratef45** [comment]
@paulodamaso I had some comments and questions, please check them.

File: timeless/templates/auth/forgot_password_post.html (hunk 3 above)
Comment (paulodamaso) [reply timeless/templates/auth/forgot_password_post.html]: @ammaratef45 When we get here we did not have any error, I've swapped the if and it is right now.

**Comment by paulodamaso** [comment]
@ammaratef45 Corrections done, please take a look

**Comment by ammaratef45** [comment]
@paulodamaso The test fails now, can you have a look?

**Comment by paulodamaso** [comment]
@ammaratef45 I can't fix this test with the current design, so I've marked to skip it and left an puzzle to correct the design, please take a look

**Comment by ammaratef45** [comment]
@paulodamaso There are some trailing whitespaces(see codeclimate), could you fix them?

**Comment by paulodamaso** [comment]
@ammaratef45 Yes I can :) Done!

**Comment by ammaratef45** [comment]
@paulodamaso Thanks.

**Comment by ammaratef45** [comment]
@rultor merge it.

**Comment by emilianodellacasa** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job was finished in 32 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @ammaratef45[/z](https://www.0crat.com/u/ammaratef45), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#372](https://github.com/timelesslounge/timelessis/issues/372) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa)

**Comment by ypshenychka** [comment]
@ammaratef45  According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

Only one major issue was found during code review. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by ammaratef45** [comment]
> @ammaratef45 According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
> 
> > The code reviewer found at least three problems in the code.
> > Comments were mostly about design problems, not cosmetic issues.
> 
> Only one major issue was found during code review.
> Please confirm that you'll try to find at least three major problems while future reviews.

@ypshenychka confirm

**Comment by ypshenychka** [comment]
@ammaratef45 thanks

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +20 point(s) just awarded to @ammaratef45[/z](https://www.0crat.com/u/ammaratef45)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**